src/main/java/frc/robot/utils/DashboardDataContainer.java

```diff
@@ -52,7 +56,17 @@ public DashboardDataContainer() {
         // Command groups data
         putData("CommandGroup/Auto Shoot", new AutoShoot());
         putData("CommandGroup/Collect Cell", new CollectCell());
-        putData("CommandGroup/Collect From Feeder", new CollectFromFeeder());
+        putData("CommandGroup/Collect From Feeder", new CollectFromFeeder()); */
+
+        //added for testing:
+        putNumber("Shooter/Shooting velocity setpoint", ShooterVelocity.kDefault.getVelocity());
+        putData("Shooter/Set cheesy shooting velocity", new CheesySetShooterVelocity(() -> getNumber("Shooter/Shooting velocity setpoint", 0)));
+        putData("Shooter/Set shooting velocity", new SetShooterVelocity(() -> getNumber("Shooter/Shooting velocity setpoint", 0)));
+        putData("Shooter/Enable tuning", new StartEndCommand(shooter::enableTuning, shooter::disableTuning));
+        putNumber("Shooter/Override Power", 0);
+        putData("Shooter/Override", new OverrideCommand(shooter,
+            () -> getNumber("Shooter/Override Power", 0)));
+        putData("Shooter/Set shooting power", new MoveMovableSubsystem(shooter, () -> getNumber("Shooter/Shooting velocity setpoint", 0)));
```

Comment:
Why we need override and set power? they both move the motor by a smart dashboard value

Reply:
I wanted to check if the override method works... it's important also for the other command group checks